utils/pullrequest.js

```diff
@@ -35,56 +35,60 @@ const pr = {
   TravisBuddy :green_heart:`,
 };
 
```

Comment:
You should take a look at this file
I've moved everything inside an async function in order to refactor those nested callbacks

Reply:
Yup, looks good